Changed the UI of project card

AKC068 opened the pull request:
As per provided reference link, I did all the changes. Please review once.
I've also provided a demo video that how it is working in last comment.

AKC068 commented:
What should I change in this according to you?

anushbhatia reviewed (changes requested):
![image](https://user-images.githubusercontent.com/40017559/121806136-520c3c80-cc6c-11eb-9fb2-cb90eaa3ff84.png)
Add appropriate padding in this so that cards won't stick together.

AKC068 commented:
@anushbhatia sure...

AKC068 commented:
I've done all the changes as per requested and required.
- Added padding to cards.
- Added extra animation on hover.
- Added AOS animation also.
- And these cards are responsive in every view and working properly.

@anushbhatia please once review all the changes and let me know if it needs more enhancement or anything different.
@smaranjitghose

smaranjitghose commented on assets/css/project.css:
```diff
@@ -35,7 +35,17 @@ html[data-theme = 'dark']
 body{
```
what is the `.swp` file for?

AKC068 replied:
I'm checking.
I don't know how it was added from my side

AKC068 replied:
@smaranjitghose its done.

anushbhatia reviewed (changes requested):
![image](https://user-images.githubusercontent.com/40017559/122458025-f4118900-cfcc-11eb-9129-5f5655467b84.png)
Try creating a new PR cause this might be having bugs with it.

AKC068 commented:
ok

smaranjitghose commented:
@AKC068 as requested by @anushbhatia to continue in another PR, I am closing this one

AKC068 commented:
@smaranjitghose can you please assign me issue #743 as per requested.
